<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyRoleDialog</class>
 <widget class="QDialog" name="ModifyRoleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改角色信息</string>
  </property>
  <widget class="QComboBox" name="cbb_IsFlowCopy">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px; 
background-color: rgb(255, 255, 255);
color: rgb(78, 160, 201);</string>
   </property>
   <item>
    <property name="text">
     <string>false</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>true</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(78, 160, 201);
font: 10pt &quot;Agency FB&quot;;
border-style: hidden;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;能否绕过安全检测：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_RoleName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(78, 160, 201);
font: 10pt &quot;Times New Roman&quot;;
border-style: hidden;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbb_IsLogin">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px; 
background-color: rgb(255, 255, 255);
color: rgb(78, 160, 201);</string>
   </property>
   <item>
    <property name="text">
     <string>false</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>true</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cbb_CreatRole">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px; 
background-color: rgb(255, 255, 255);
color: rgb(78, 160, 201);</string>
   </property>
   <item>
    <property name="text">
     <string>false</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>true</string>
    </property>
   </item>
  </widget>
  <widget class="QDateTimeEdit" name="dte_authdate">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>194</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px; 
background-color: rgb(255, 255, 255);
color: rgb(78, 160, 201);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(78, 160, 201);
font: 10pt &quot;Agency FB&quot;;
border-style: hidden;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;继承：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(78, 160, 201);
font: 10pt &quot;Agency FB&quot;;
border-style: hidden;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;超级管理员：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_connect">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px; 
background-color: rgb(255, 255, 255);
color: rgb(78, 160, 201);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(78, 160, 201);
font: 10pt &quot;Agency FB&quot;;
border-style: hidden;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;创建角色：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbt_Cancle">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);
background-color: rgb(78, 160, 201);
border-radius: 5px; 
</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbb_IsSkipAuth">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px; 
background-color: rgb(255, 255, 255);
color: rgb(78, 160, 201);</string>
   </property>
   <item>
    <property name="text">
     <string>false</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>true</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(78, 160, 201);
font: 10pt &quot;Agency FB&quot;;
border-style: hidden;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;角色名：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(78, 160, 201);
font: 10pt &quot;Agency FB&quot;;
border-style: hidden;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;并发链接数：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbb_CreatDB">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px; 
background-color: rgb(255, 255, 255);
color: rgb(78, 160, 201);</string>
   </property>
   <item>
    <property name="text">
     <string>false</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>true</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pbt_OK">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>340</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);
background-color: rgb(78, 160, 201);
border-radius: 5px; 
</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(78, 160, 201);
font: 10pt &quot;Agency FB&quot;;
border-style: hidden;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;创建数据库：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbb_IsInherit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px; 
background-color: rgb(255, 255, 255);
color: rgb(78, 160, 201);</string>
   </property>
   <item>
    <property name="text">
     <string>false</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>true</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(78, 160, 201);
font: 10pt &quot;Agency FB&quot;;
border-style: hidden;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;过期时间：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbb_IsSuper">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px; 
background-color: rgb(255, 255, 255);
color: rgb(78, 160, 201);</string>
   </property>
   <item>
    <property name="text">
     <string>false</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>true</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(78, 160, 201);
font: 10pt &quot;Agency FB&quot;;
border-style: hidden;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;能否发起流复制：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(78, 160, 201);
font: 10pt &quot;Agency FB&quot;;
border-style: hidden;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;能否登录：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
